src/Entity/Resources/AAKResource.php

```diff
@@ -11,200 +11,235 @@
  * Extbooking.aakresources.
  *
  * @ORM\Table(name="ExtBooking.AAKResources")
+ *
  * @ORM\Entity
  */
 class AAKResource
 {
     /**
      * @Groups({"resource", "minimum"})
+     *
```

Comment:
Are these (almost) blank lines a result of updated coding standards or personal taste? I think they don't add any value, e.g. improved readability, and only make the files longer.

Reply:
Updated coding standards.
Maybe this is the issue: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6733

Reply:
Seems to be an issue with php-cs-fixer: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6733
Have used solution from: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6733#issuecomment-1569919413